test/test_custom_ops.py

```diff
@@ -2117,6 +2117,47 @@ def _(x, y):
         self.assertEqual(z, x + y)
         self.assertTrue(cpu_called)
 
+    def test_mutated(self):
+        @torch.library.custom_op(
+            "_torch_testing::numpy_sin_inplace", mutated_args={"x"}, device_types="cpu"
```

Comment:
Why cpu only here?

Reply:
otherwise np.sin(x_np, out=x_np) doesn't actually update the original input